test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -407,6 +519,46 @@ public void ConfigGetCommand_NullGetLogger_Fail()
             Assert.Throws<ArgumentNullException>(() => ConfigGetRunner.Run(args, null));
         }
 
+        [Fact]
+        public void ConfigSetCommand_WithInvalidConfigKey_ThrowsCommandException()
+        {
+            // Arrange & Act
+            using var testInfo = new TestInfo("NuGet.Config");
+            var key = "InvalidConfigKey123";
+            var value = "https://TestRepo2/ES/api/v2/package";
+            var filePath = Path.Combine(testInfo.WorkingPath, "NuGet.Config");
+
+            var result = CommandRunner.Run(
+                DotnetCli,
+                Directory.GetCurrentDirectory(),
+                $"{XplatDll} config set {key} {value} --configfile {filePath}",
+                waitForExit: true);
+            var expectedError = string.Format(CultureInfo.CurrentCulture, Strings.Error_ConfigSetInvalidKey, key);
+
+            // Assert
+            DotnetCliUtil.VerifyResultFailure(result, expectedError);
+        }
+
+        [Fact]
+        public void ConfigSetCommand_WithNullArgs_ThrowsArgumentNullException()
+        {
+            // Arrange
+            var log = new TestCommandOutputLogger();
+
+            // Act & Assert
+            Assert.Throws<ArgumentNullException>(() => ConfigSetRunner.Run(args: null, () => log));
+        }
+
+        [Fact]
+        public void ConfigSetCommand_WithNullGetLogger_ThrowsArgumentNullException()
+        {
+            // Arrange
+            var args = new ConfigSetArgs();
+
+            // Act & Assert
+            Assert.Throws<ArgumentNullException>(() => ConfigSetRunner.Run(args, getLogger: null));
+        }
```

Comment:
Are these two test names mixed up? It looks like the first test tests for a null logger and the second test tests for null args but the names of the tests suggest otherwise.

Reply:
In the new Theory test, I tried to make it more evident which arguments are null